modules/06 Angular/Assignments/15/answer/src/app/app.component.html

```diff
@@ -0,0 +1,13 @@
+<app-header></app-header >
+<div id="container" [className]=" (bigPicSrc!=='')? 'blur' : '' ">
```

Comment:
```suggestion
<div id="container" [class.blur]="bigPicSrc">
```

Reply:
Got your answer and implement it on the html but also in the app.component.ts instead of bigPicSrc="" ---> bigPicSrc=null at the closeBigPic() function